cockatrice/src/tab_game.cpp

```diff
@@ -309,6 +318,7 @@ TabGame::TabGame(TabSupervisor *_tabSupervisor, GameReplay *_replay)
     sayLabel(0),
     sayEdit(0)
 {
+    MainWindow = new QMainWindow;
```

Comment:
MainWindow is added to a layout in line 449, and then the layout is applied to TabGame.
As stated on http://doc.qt.io/qt-4.8/layout.html#tips-for-using-layouts , the layout will take care of reparenting all the widgets to TabGame.